isisd/isis_vty.c

```diff
@@ -2085,6 +2100,8 @@ void isis_vty_init(void)
 	install_element(INTERFACE_NODE, &isis_hello_padding_cmd);
 	install_element(INTERFACE_NODE, &no_isis_hello_padding_cmd);
 
+	install_element(INTERFACE_NODE, &isis_threeway_adj_cmd);
+
```

Comment:
Same. Only one command is added while the command could also disable 3 way handshake.

If default is to enable 3 way handshake, perhaps, the command should be rename `no_isis_threeway_adj_cmd' and adjust the DEFUN accordingly

Reply:
I think it's fine, no need to change this, with combined enable/disable commands we usually leave the 'no' off of the front